libs/labelbox/tests/integration/test_model_chat_evaluation_ontology.py

```diff
@@ -0,0 +1,86 @@
+from labelbox import OntologyBuilder, Tool
+from labelbox import MediaType
+
+
+def test_create_model_chat_evaluation_ontology(client, rand_gen):
+    ontology_name = f"test-model-chat-evaluation-ontology-{rand_gen(str)}"
+    ontology_builder = OntologyBuilder(tools=[
+        Tool(tool=Tool.Type.MESSAGE_SINGLE_SELECTION,
+             name="model output single selection"),
+        Tool(tool=Tool.Type.MESSAGE_MULTI_SELECTION,
+             name="model output multi selection"),
+        Tool(tool=Tool.Type.MESSAGE_RANKING, name="model output multi ranking"),
+    ],)
+    ontology = None
+
+    try:
+        ontology = client.create_model_chat_evaluation_ontology(
+            ontology_name, ontology_builder.asdict())
+        assert ontology
+        assert ontology.name == ontology_name
+        assert len(ontology.tools()) == 3
+        for tool in ontology.tools():
+            assert tool.schema_id
+            assert tool.feature_schema_id
+
+    finally:
+        if ontology:
+            client.delete_unused_ontology(ontology.uid)
+
+
+def test_ontology_create_feature_schema(client, rand_gen):
+    try:
```

Comment:
Can any of this be pulled out into a fixture? I think that's a better place to create/cleanup this kind of test data instead of this try/catch/finally structure.

Reply:
not sure because the test itself is testing creation of ontology, but possibly in a follow up PRs where I add project and linking ontology to project